packages/salesforcedx-vscode-soql/src/commands/soqlFileCreate.ts

```diff
@@ -17,16 +17,16 @@ export async function soqlOpenNew(): Promise<void> {
   telemetryService.sendCommandEvent('soql_builder_open_new', process.hrtime());
 
   if (vscode.workspace) {
-    // create untitled file
-    const doc = await vscode.workspace.openTextDocument({
-      language: 'soql',
-      content: ''
+    const fileName = 'untitled.soql';
```

Comment:
If I run the "Create Query in SOQL..." command a second time before saving, nothing happens. I guess it is because it is always the same filename here.

If we are OK with this behavior, fine. If not, you might want to extract that hard-coded filename into a function to generate a unique file for the current workspace.

Reply:
I messed with this a bit, and the behavior of untitled files with Custom Editors even with a numeric file sequence is not any better.  Check out this movie.  I'm a little dumbfounded...and ignorant still about much of the vscode OS along with the behavior of Custom Editors.  If i use the openTextDocument() api, then the Save As doesn't behave as we expect.

https://user-images.githubusercontent.com/599418/112670483-9125b100-8e26-11eb-8d9e-c8c1aa3571d5.mov

Reply:
At this point, I say we go out with this behavior and see what people think.  At least we are functioning again.